### DISCUSSION TITLE
Rückmeldung nach Schwedentour / Feedback after Sweden tour

### DISCUSSION THREAD

**Discussion by Basti32** [devemux86/cruiser#619] [Q&A] [answered]
Hallo, ich habe die App jetzt ca 10 Tage auf rund 2000 km testen können und es gab keinen Grund auf eine andere App zu wechseln. Somit nutzte ich Cruiser vom 1. bis zum letzten Kilometer. Bin total begeistert, wie gut alles verlief. 👍 
Da ich in Schweden oft ohne Netz war, hat sich die offline Routen sehr bewährt. Was mir noch fehlt, ist die unterschiedliche Farbdarstellung der Route in Bezug auf den Fahrbahnbelag und eine gute Webseite für die erste Routenplanung. Danach übernimmt so wieso die Tour die Routenführung. Viele Grüße Basti PS: Da bin ich ja gespannt, was ihr da noch verbessern wollt. (Cruiser 5)

Transaltor DeepL

Hello, 
I have now been able to test the app for about 10 days on about 2000 kilometres and there was no reason to switch to another app. I used Cruiser from the first to the last kilometre. I'm absolutely thrilled with how well everything went. 👍 
As I was often without a network in Sweden, the offline routes proved very useful. What I'm still missing is the different colour display of the route in relation to the road surface and a good website for the initial route planning. After that, the tour takes over the route guidance. 
Best regards 
Basti 
PS: I'm looking forward to seeing what you want to improve. (Cruiser 5)

**Comment by devemux86** [comment, marked as answer]
@Basti32 

Thanks for the kind words!
You can change the route color in "Settings | Routing | Appearance | Colors".

Vielen Dank für die freundlichen Worte!
Sie können die Farbe der Route unter "Einstellungen | Routing | Aussehen | Farben" ändern.

**Comment by Ciutnik** [comment]
> different color display of the route in relation to the road surface 

My games with map themes have shown that to some extent it is possible. I already distinguish "stones" or "unpaved" roads with a colored dashed lines/simple paterns. Because, as reality shows, sometimes the road category does not correspond to the expected surface. There is even a "smoothness" tag but unfortunately probably rarely used in mapping.

<img src="https://github.com/user-attachments/assets/3703e9d8-1862-49c7-a486-ebfefb5c486b" width="50%"/>

**Reply by devemux86** [reply]
@Ciutnik 

Very nice! This rendering works much better in the new map engine.

If you have specific suggestions for the Motorider theme please write them there:
- https://github.com/devemux86/cruiser/discussions/612
